### PR TITLE
Updated all atlassian docker images versions

### PR DESCRIPTION
**Pull request by robertocuervo**
Signed-off-by: robertocuervo <cuervo_alvarez_roberto@hotmail.com>

### CODE REVIEW

**Comment by robertocuervo** [comment]
Thanks for your repository! I just updated the images versions.

**Comment by omidraha** [comment]
Hi,
Also please update  in `README.md` file too.
Thanks.

**Comment by robertocuervo** [comment]
Updated atlassian images versions in Readme.md

File: README.md [hunk 1]
```diff
@@ -69,22 +69,23 @@ How to use:
...
 4. Set `DNS` according to the above `DOMAIN` value, on somewhere that you want to connect to host of `docker-compose`:
 
 
-    ``` 
+    ```
     $ vim /etc/hosts
         127.0.0.1 jira.example.com www.jira.example.com
         127.0.0.1 wiki.example.com www.wiki.example.com
         127.0.0.1 bitbucket.example.com www.bitbucket.example.com
+        127.0.0.1 bamboo.example.com www.bamboo.example.com
```
Comment (omidraha) [inline README.md]: There is no `bamboo`.

**Comment by robertocuervo** [comment]
Removed unusued bamboo link in Readme

**Comment by robertocuervo** [comment]
...and updated compose file version from 3 to 3.4

File: docker-compose.yml [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
   bitbucket:
-    image: atlassian/bitbucket-server:4.14
+    image: atlassian/bitbucket-server:5.10.1
     restart: always
     links:
       - database
     volumes:
       - bitbucket-data:/var/atlassian/application-data/bitbucket
 
+  bamboo:
```
Comment (omidraha) [inline docker-compose.yml]: You removed bamboo from readme, but still there is some code related to it in this file.
Review whole the file.

**Comment by robertocuervo** [comment]
... and removed unused bamboo service in docker compose file.

**Pull request merged**